lib/rgeo/feature/factory.rb

```diff
@@ -99,7 +97,7 @@ def parse_wkb(str)
       # supported. If both Z and M coordinates are supported, Z should
       # be passed first.
 
-      def point(x, y, *extra)
+      def point(_x_coord, _y_coord, *_extra)
```

Comment:
@keithdoggett shouldn't we raise here rather than returning nil ?

Reply:
hmm. I guess so, although this should never be called in general, but yes raising a `NotImplementedError` would probably be better

Reply:
@BuonOmo & @keithdoggett what should the error message say?

Reply:
@thestelz don't bother yet, I think this should be part of its own PR for better traceability! This is a breaking change!